Source/UnrealGT/Private/GTRandomColorGenerator.cpp

```diff
@@ -0,0 +1,45 @@
+#include "GTRandomColorGenerator.h"
+
+#include "Math/Color.h"
+
+TArray<int> MakeSequentialIndexArray(const int FirstIndex, const int LastIndex)
+{
+    TArray<int> arr;
+    arr.Reserve(255);
```

Comment:
```suggestion
    arr.Reserve(LastIndex - FirstIndex + 1);
```
?

Reply:
Good call. This was just copy-pasted over to this new function, but now I've made the change you suggest.